src/constant/theme-colors.js

```diff
@@ -12,6 +12,15 @@ const themeColors = {
     'orange-default': 'rgba(255, 176, 32, 1)',
     'orange-light': 'rgba(255, 245, 226, 1)',
     'orange-DBK': 'rgba(255,124,96, 1)',
+
+    'sonic-btn': 'rgba(33, 78, 129, 0.16)',
+    'sonic-btn-foreground': 'rgba(33, 78, 129, 1)',
+    'sonic-card': 'rgba(255, 255, 255, 1)',
+    'sonic-card-foreground': 'rgba(0, 0, 0, 1)',
+    'sonic-card-border': 'rgba(80, 97, 121, 0.1)',
+    'sonic-background': 'rgba(245, 245, 245, 1)',
+    'sonic-foreground': 'rgba(0, 0, 0, 1)',
+
```

Comment:
can you make both light and dark to the end of object like below?
for separate core theme variables and external theme variables
![image](https://github.com/user-attachments/assets/63016225-38c5-4512-ae9c-424f8807c87e)